src/desert/_make.py

```diff
@@ -213,10 +213,7 @@ def field_for_schema(
     if default is not marshmallow.missing:
         desert_metadata.setdefault("default", default)
         desert_metadata.setdefault("allow_none", True)
-        if not desert_metadata.get(
-            "required"
-        ):  # 'missing' must not be set for required fields.
-            desert_metadata.setdefault("missing", default)
+        desert_metadata.setdefault("missing", default)
```

Comment:
This seems like the potentially controversial change here.  If we want to worry about refactoring this first and address the coverage later, I get it.  Or, perhaps I'm mismerembering where the 'mush' to be reworked was.

Reply:
Mind doing this one as a separate PR?